CHANGES.md

```diff
@@ -11,6 +11,7 @@ Unreleased changes refer to our current [master branch](https://github.com/openc
 
 ### Fixes
 * Fixed `Too many connections to ldap` when openchange runs on samba as member of a domain.
+* Fixed `Mark All as Read` feature regression bug introduced by 7737bdf6
```

Comment:
imho, the `changes.md` file should be targeting end users (non-developers) so it doesn't make sense to include hash values of commit. Just keep descriptions simple even if they are not accurate.